### PR TITLE
Android/2.0.0 blog post

### PR DESCRIPTION
**Pull request by skraynick**

### CODE REVIEW

File: _posts/2016-07-18-android_sdk_2.0.0_is_here.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+comments: true
+tags: beacon SDK Android SensorbergSDK
+---
+  
+### We have released [version 2.0.0 of the Sensorberg Android SDK](https://github.com/sensorberg-dev/android-sdk)  
+
+# Improvements
+- Android 6 permissions now apply.  You can view the specifics on the blogpost for [Android 6.](https://developer.sensorberg.com/2016/07/Be-Ready-For-Android6-Permissions/)
+- Proguard Configuration was added. Find more information [here.](https://developer.sensorberg.com/2016/07/Proguard-support-for-Android-SDK/)
+
```
Comment (mrsasha) [inline _posts/2016-07-18-android_sdk_2.0.0_is_here.md]: I would just add that the implementation slightly changed - different class and init method, and no need for a bootstrapper any more.

File: _posts/2016-07-18-android_sdk_2.0.0_is_here.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+comments: true
+tags: beacon SDK Android SensorbergSDK
+---
+  
+### We have released [version 2.0.0 of the Sensorberg Android SDK](https://github.com/sensorberg-dev/android-sdk)  
+
+# Improvements
+- Android 6 permissions now apply.  You can view the specifics on the blogpost for [Android 6.](https://developer.sensorberg.com/2016/07/Be-Ready-For-Android6-Permissions/)
+- Proguard Configuration was added. Find more information [here.](https://developer.sensorberg.com/2016/07/Proguard-support-for-Android-SDK/)
+- Implementation of the sdk has also changes to see an example please the dev-app application class and read our implemention [guide](http://sensorberg-dev.github.io/android/)
```
Comment (mrsasha) [inline _posts/2016-07-18-android_sdk_2.0.0_is_here.md]: this doesn't sound like english ;)

Comment (skraynick) [inline _posts/2016-07-18-android_sdk_2.0.0_is_here.md]: it is serbo-english. lol :D

Comment (mrsasha) [inline _posts/2016-07-18-android_sdk_2.0.0_is_here.md]: It's called serblish...

**Pull request merged**